src/messages/Lifecycle.ts

```diff
@@ -100,7 +100,9 @@ const envDecoder: Decoder<LabeledEnvInit> = labeledDecoder(
 export { startedDecoder, envDecoder };
 
 /**
- * Handles new environmental data events.
+ * Handles new environmental data events. These should be configured
+ * in the same tick that {@link Client.start} is called, as the host
```

Comment:
Is 'tick' lifecycle vernacular? I don't think I've heard this expression before.  You're saying in the life cycle loop correct?

Reply:
It's intended to be JS vernacular, but I think it's mostly used in node. I'm trying to say it needs to happen in the same stack of functions. It can't be in something that happens asynchronously after `Client.start`, even if it's called right away.

Initially I just said to call it before `Client.start`.  Technically what I have is more precise, but I worried it could be confusing, and it seems like it was. I think I will just go back to the "before" wording.